script.sh

```diff
@@ -4,6 +4,15 @@ set -e
 NOW_DEPLOY_OPTIONS=" --no-clipboard"
 NOW_AUTH=""
 
+# Is this a public deployment?
+if [ "$PLUGIN_PUBLIC" == "true" ]
+then
+  echo "> adding option --public"
```

Comment:
What kind of precision are you looking for here? It seems obvious enough to me that the option corresponds to the `now` CLI given that this is a Drone plugin for precisely that.  I'm happy to modify but I'm not clear on what other information you'd like to include here.

Reply:
> For every deployment done under the OSS plan, this needs to be set to true.
It could be good to inform the user that if he has a OSS plan, his source code and logs will be public, for example.